eng/scripts/Set-VcpkgWriteModeCache.ps1

```diff
@@ -1,3 +1,5 @@
+#cspell: ignore azblob

```

Comment:
But there is no `azblob` in this file.

Reply:
Nevermind... I somehow didn't expand the file all the way down to the last couple lines :P